app.js

```diff
@@ -1,7 +1,23 @@
 import config from './config/config.json';
 import { User, Product } from './models';
+import { DirWatcher, Importer } from './file-handling';
 
 console.log(config.name);
 
 const user = new User();
-const product = new Product();
\ No newline at end of file
+const product = new Product();
+
+const dirWatcher = new DirWatcher();
+dirWatcher.watch("./data", 5000);
+
+const importer = new Importer();
+
+dirWatcher.on('changed', (path) => {
+    importer.import(path).then((json) => {
```

Comment:
please handle the situation when file is not possible to process (incorrect format) or it's a folder